client/state/themes/current-theme/test/reducer.js

```diff
@@ -12,7 +12,7 @@ import {
 	SERIALIZE,
 	DESERIALIZE,
 	SERVER_DESERIALIZE
-} from 'state/action-types';
+} from '../../../action-types';
```

Comment:
Feels easier to manage as `state/action-types`, especially if we reorganize or further nest parts of the tree.